pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -97,39 +98,22 @@ var _ = Describe("ProjectService", func() {
 	})
 
 	Describe("enabled", func() {
-
 		When("component toggle label is set to Truthy value", func() {
 			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelComponentEnabled: "true",
-				}
+				k8sconf.Disabled = true
 			})
 
 			It("returns true", func() {
-				Expect(projectService.enabled()).To(BeTrue())
-			})
-		})
-
-		When("component toggle label is set to Falsy value", func() {
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelComponentEnabled: "false",
-				}
-			})
-
-			It("returns false", func() {
 				Expect(projectService.enabled()).To(BeFalse())
 			})
 		})
 
-		When("component toggle label is set to any string value not representing a boolean value (one of: 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False.)", func() {
+		When("component toggle extension to set false value", func() {
```

Comment:
The configuration map may contain values that aren't bool.. eg. "true" , 1, TRUE. It'd make sense to respect and handle those "truthy" values I think.

Reply:
again, we no longer look at maps, we only look at the configuration struct which is type safe so if those values are set wrong they will be either caught by marshalling or validation, this is not the place to check for any of it.